src/Stratis.Bitcoin/P2P/Peer/NetworkPeerServer.cs

```diff
@@ -104,7 +92,6 @@ public NetworkPeerServer(Network network, IPEndPoint localEndpoint, IPEndPoint e
             this.dateTimeProvider = dateTimeProvider;
             this.networkPeerFactory = networkPeerFactory;
 
-            this.AllowLocalPeers = true;
             this.InboundNetworkPeerConnectionParameters = new NetworkPeerConnectionParameters();
 
             this.LocalEndpoint = Utils.EnsureIPv6(localEndpoint);
```

Comment:
Two 'param name' tags are not corresponding the variable names used in this constructor: 'localEndpoint' and 'externalEndpoint'

Reply:
I don't understand, please explain

Reply:
The xml doc refers param names that do not exist. Someone changed names of the constructor's variables and forgot about the xml doc changes.

Reply:
aah, thank you i see it now, will fix